test/StringProcessing/Parse/forward-slash-regex.swift

```diff
@@ -121,6 +121,7 @@ do {
 } // expected-error {{expected expression after operator}}
 
 _ = /x/??/x/ // expected-error {{'/' is not a postfix unary operator}}
+// expected-error@-1 2 {{annot use optional chaining on non-optional value of type 'Regex<Substring>'}}
```

Comment:
WDYT, @hamishknight ?

Reply:
I think this makes sense, as it'll be parsed as `(/x/)?? / (x/)`

Reply:
Thanks!

Reply:
I need to add `c` I dropped by mistake from `cannot` 🤦